### PR TITLE
VIDTC NFT's for Binance Smart Chain (3 addresses)

### PR DESCRIPTION
**Pull request by Sjazzz**
Added the info and logo's for Binance Smart Chain NFT's following the ERC721 standard.

Thanks again team TrustWallet!

### CODE REVIEW

File: blockchains/smartchain/assets/0x0567D7fad179dCd2a89508B77cCe93E26f8998F7/info.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "VIDTC Luxury goods",
+    "website": "https://about.v-id.org",
+    "short_description": "NFTs for watches, fashion & jewelry",
+    "explorer": "https://bscscan.com/address/0x0567d7fad179dcd2a89508b77cce93e26f8998f7"
```
Comment (Iamdeadlyz) [inline blockchains/smartchain/assets/0x0567D7fad179dCd2a89508B77cCe93E26f8998F7/info.json:5]: Please update it to `https://bscscan.com/token/0x0567d7fad179dcd2a89508b77cce93e26f8998f7`

File: blockchains/smartchain/assets/0x7A2172F3728962F6B952a1Cc1AD794DACbf9f3DA/info.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "VIDTC Company stocks",
+    "website": "https://about.v-id.org",
+    "short_description": "NFTs for equity and shared ownership",
+    "explorer": "https://bscscan.com/address/0x7a2172f3728962f6b952a1cc1ad794dacbf9f3da"
```
Comment (Iamdeadlyz) [inline blockchains/smartchain/assets/0x7A2172F3728962F6B952a1Cc1AD794DACbf9f3DA/info.json:5]: Update this as well to `https://bscscan.com/token/0x7a2172f3728962f6b952a1cc1ad794dacbf9f3da`

File: blockchains/smartchain/assets/0x7B7D3B71A12906270b1b4c187C75f45188E9aEf6/info.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "VIDTC Precious metals and stones",
+    "website": "https://about.v-id.org",
+    "short_description": "NFTs for valuables like diamonds or gold",
+    "explorer": "https://bscscan.com/address/0x7b7d3b71a12906270b1b4c187c75f45188e9aef6"
```
Comment (Iamdeadlyz) [inline blockchains/smartchain/assets/0x7B7D3B71A12906270b1b4c187C75f45188E9aEf6/info.json:5]: This one as well to `https://bscscan.com/token/0x7b7d3b71a12906270b1b4c187c75f45188e9aef6`

**Review by Iamdeadlyz — CHANGES_REQUESTED** [review]

**Comment by catenocrypt** [comment]
These All these 3 tokens have very low circulation on BSC, a min. of 500 holders is the limit for inclusion.
Are these tokens active on other chain as well (like eth)?

**Comment by Sjazzz** [comment]
> These All these 3 tokens have very low circulation on BSC, a min. of 500 holders is the limit for inclusion.
> Are these tokens active on other chain as well (like eth)?

Hi, these are new NFT contracts that will connect to the VIDT Datalink contracts at ...
https://etherscan.io/token/0xfeF4185594457050cC9c23980d301908FE057Bb1
and
https://bscscan.com/token/0x3f515f0a8e93f2e2f891ceeb3db4e62e202d7110

Currently there are about 5,300 holders.

Thanks again!

**Review by Iamdeadlyz — APPROVED** [review]

**Pull request merged**